src/main/java/com/fastcampus/projectboard/config/ThymeleafConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.fastcampus.projectboard.config;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.thymeleaf.spring5.templateresolver.SpringResourceTemplateResolver;
+
+@Configuration
+public class ThymeleafConfig {
+
+    @Bean
+    public SpringResourceTemplateResolver thymeleafTemplateResolver(
+            SpringResourceTemplateResolver defaultTemplateResolver, // 기본 템플릿 resolver
+            Thymeleaf3Properties thymeleaf3Properties // 사용자 정의 프로퍼티 클래스
+    ) {
+        defaultTemplateResolver.setUseDecoupledLogic(thymeleaf3Properties.isDecoupledLogic()); // 사용자 정의에 따라 로직 사용 여부 결정
```

Comment:
주석이 다소 모호하게 느껴집니다.

```suggestion
        defaultTemplateResolver.setUseDecoupledLogic(thymeleaf3Properties.isDecoupledLogic()); // 사용자 정의에 따라 thymeleaf decoupled logic 사용 여부 결정
```

Reply:
사실 이 정도면 코드를 읽고 이해할 수 있다고 볼 수도 있습니다. decoupled logic이 무엇인지를 설명할게 아니라면요.

```suggestion
        defaultTemplateResolver.setUseDecoupledLogic(thymeleaf3Properties.isDecoupledLogic());
```